<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormItens</class>
 <widget class="QWidget" name="FormItens">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1756</width>
    <height>1222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Itens</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="header">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="headerVLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="headerRowA">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_cliente">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_cliente">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Cliente:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_cliente_val">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_user">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_user">
            <property name="text">
             <string>Utilizador:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_user_val">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="headerSpacerA">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="headerRowB">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_ano">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_ano">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Ano:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_ano_val">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_num">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_num">
            <property name="text">
             <string>Nº Orçamento:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_num_val">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_ver">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_ver">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Versão:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_ver_val">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="headerSpacerB">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="form_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="layout_codigo">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_codigo">
          <property name="text">
           <string>Código</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_codigo">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="6">
       <layout class="QHBoxLayout" name="layout_profundidade">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_profundidade">
          <property name="text">
           <string>Profundidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_profundidade">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_item">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <layout class="QHBoxLayout" name="layout_qt">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_qt">
          <property name="text">
           <string>Qt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_qt">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>24</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_item">
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <spacer name="rowSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>776</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="8">
       <layout class="QHBoxLayout" name="layout_und">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_und">
          <property name="text">
           <string>Und</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_und">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>und</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="layout_altura">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_altura">
          <property name="text">
           <string>Altura</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_altura">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2" colspan="7">
       <widget class="QTextEdit" name="edit_descricao">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Descrição do item...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lbl_descricao">
        <property name="text">
         <string>Descrição</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <layout class="QHBoxLayout" name="layout_largura">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_largura">
          <property name="text">
           <string>Largura</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_largura">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Inserir Novo Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Gravar Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="text">
        <string>Eliminar Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_expand">
       <property name="text">
        <string>Expandir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_collapse">
       <property name="text">
        <string>Colapsar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_up">
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_dn">
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="table">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
